postgres-driver/loadbalancer.go

```diff
@@ -0,0 +1,59 @@
+package postgresdriver
+
+import (
+	"database/sql"
+	"time"
+)
+
+const (
+	selectLoadbalancerByID = "SELECT * FROM loadbalancers WHERE lb_id = $1"
+)
+
+type dbLoadbalancer struct {
+	ID                int            `db:"id"`
+	LbID              string         `db:"lb_id"`
+	Name              sql.NullString `db:"name"`
+	CreatedAt         sql.NullTime   `db:"created_at"`
+	UpdatedAt         sql.NullTime   `db:"updated_at"`
+	RequestTimeout    sql.NullInt64  `db:"request_timeout"`
+	Gigastake         sql.NullBool   `db:"gigastake"`
+	GigastakeRedirect sql.NullBool   `db:"gigastake_redirect"`
+	UserID            sql.NullString `db:"user_id"`
+}
+
+func (lb *dbLoadbalancer) toLoadBalancer() *Loadbalancer {
+	return &Loadbalancer{
+		LbID:              lb.LbID,
+		Name:              lb.Name.String,
+		CreatedAt:         &lb.CreatedAt.Time,
+		UpdatedAt:         &lb.UpdatedAt.Time,
+		RequestTimeout:    lb.RequestTimeout.Int64,
+		Gigastake:         lb.Gigastake.Bool,
+		GigastakeRedirect: lb.GigastakeRedirect.Bool,
+		UserID:            lb.UserID.String,
+	}
+}
+
+// Loadbalancer struct handler representing a Loadbalancer
+type Loadbalancer struct {
```

Comment:
Could we change and use the Loadbalancer struct from the `repository` package instead?
This will then remove the need to do another conversion somewhere else.
Basically, it may be best for the postgres-driver package to translate from the actual db fields/types (that no other package should be aware of) into `repository` package types (which all other packages then depend on)